queue/abstract/driver/limfifottl.lua

```diff
@@ -0,0 +1,42 @@
+local fiber    = require('fiber')
+local fifottl  = require('queue.abstract.driver.fifottl')
+
+local tube = {}
+local methods = {}
+
+tube.create_space = fifottl.create_space
+
+-- start tube on space
+function tube.new(space, on_task_change, opts)
+    local state = {
+        capacity = opts.capacity or 0,
+        parent = fifottl.new(space, on_task_change, opts)
+    }
+
+    -- put task in space
+    local put = function (self, data, opts)
+        local timeout = opts.timeout or 0
+        timeout = timeout * 1000000
```

Comment:
If fiber.time64() will be changed on fiber.time(), then this multiplication will be not needed.